README.md

```diff
@@ -145,6 +145,9 @@ or if the field `test_inputs` does not contain a list, it would print:
 
 
 ## Changelog
+#### bioimageio.spec 0.4.1.post0
```

Comment:
Can we also include the change to dependencies (see https://github.com/bioimage-io/spec-bioimage-io/issues/341) before making a new release? (Can ofc be done in a separate PR)

Reply:
included it here now to avoid merge conflicts and it being a small change...